### ISSUE TITLE
CurieBLE: cannot discover advertised 16-bit service of SensorTag

### ISSUE THREAD

**Issue by sandeepmistry** [01org/corelibs-arduino101#371]
When using the scan example with the my TI SensorTag I'm not able to see the `aa80` service it advertises which is in the advertisement data it advertises. However, using LightBlue on iOS I can see the 0xaa80 service.

**Comment by SidLeung** [comment]
Logged as part of Jira 792 and under internal review.

**Comment by kitsunami** [comment]
Addressed by PR #450

**Comment by sandeepmistry** [comment]
I've tested this with the v2.0.1 RC1 release, and I see the correct behaviour. Closing.

```
Discovered a peripheral
-----------------------
Address: 68:C9:0B:06:BC:81
Local Name: CC2650 SensorTag
Service UUID's: aa80 
RSSI: -54
```
